src/content/docs/en/reference/configuration-reference.mdx

```diff
@@ -492,13 +492,17 @@ Specifies the directory in the build output where Astro-generated assets (bundle
 
 <p>
 
-**Type:** `string`<br />
+**Type:** `(string | Record.<string, string>)`<br />
 **Default:** `undefined`<br />
 <Since v="2.2.0" />
 </p>
 
 Specifies the prefix for Astro-generated asset links. This can be used if assets are served from a different domain than the current site.
 
+<p>
+**If use `string`**
+</p>
```

Comment:
```suggestion
<p>
**If this value is a `string`,**
</p>
```

If you word it like this, you will likely have to reword the following text below it to flow better.